api/app/signals/apps/api/v1/serializers/nested/type.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+from signals.apps.api.generics.serializers import SIAModelSerializer
+from signals.apps.signals.models import Type
+
+
+class _NestedTypeModelSerializer(SIAModelSerializer):
+    code = serializers.CharField(source='name', max_length=3)
```

Comment:
When will the renaming of the `name` field on the model happen? Why not do this straight away?

Reply:
Started on the outside, working in, wanted test coverage on the REST API side first.